en/deploy-tidb-from-kubernetes-kind.md

```diff
@@ -19,143 +19,77 @@ kind is a tool for running local Kubernetes clusters using Docker containers as
+- [Helm](https://helm.sh/docs/intro/install/): Helm 2 or the latest stable version of Helm 3
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl): version >= 1.12
+- [kind]: version >= 0.7.0 (the latest version recommended)
 - The value of [net.ipv4.ip_forward](https://linuxconfig.org/how-to-turn-on-off-ip-forwarding-in-linux) should be set to `1`
 
 ## Step 1: Create a Kubernetes cluster using kind
 
-First, make sure that Docker is running. Then, you can create a local Kubernetes cluster with a script in our repository. Follow the steps below:
-
-1. Clone the code:
-
-    {{< copyable "shell-regular" >}}
-
-    ``` shell
-    git clone --depth=1 https://github.com/pingcap/tidb-operator && \
-    cd tidb-operator
-    ```
-
-2. Run the script and create a local Kubernetes cluster:
-
-    {{< copyable "shell-regular" >}}
-
-    ``` shell
-    hack/kind-cluster-build.sh
-    ```
-
-    > **Note:**
-    >
-    > By default, this script starts a Kubernetes cluster of the 1.12.8 version, with six nodes in the cluster and for each node the number of mount points is 9. You can configure these items by startup options.
-    >
-    > {{< copyable "shell-regular" >}}
-    >
-    > ```shell
-    > hack/kind-cluster-build.sh --nodeNum 2 --k8sVersion v1.14.6 --volumeNum 3
-    > ```
-
-3. To connect the local Kubernetes cluster, set the default configuration file path of kubectl to `kube-config`.
-
-    {{< copyable "shell-regular" >}}
-
-    ```shell
-    export KUBECONFIG="$(kind get kubeconfig-path)"
-    ```
-
-4. Verify whether the Kubernetes cluster is on and running:
-
-    {{< copyable "shell-regular" >}}
-
-    ``` shell
-    kubectl cluster-info
-    ```
-
-    The output is like this:
+Refer to [`kind` Quick Start](https://kind.sigs.k8s.io/docs/user/quick-start) to install `kind` and create a cluster.
 
-    ``` shell
-    Kubernetes master is running at https://127.0.0.1:50295
-    KubeDNS is running at https://127.0.0.1:50295/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
-    ```
-
-5. Check the `storageClass` of the cluster:
+The following is an example of using `kind` v0.8.1:
 
-    {{< copyable "shell-regular" >}}
+```
+curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.8.1/kind-$(uname)-amd64
+chmod +x ./kind
+./kind create cluster
+```
 
-    ``` shell
-    kubectl get storageClass
-    ```
+Check whether the cluster is successfully created:
 
-    The output is like this:
+{{< copyable "shell-regular" >}}
 
-    ``` shell
-    NAME                 PROVISIONER                    AGE
-    local-storage        kubernetes.io/no-provisioner   7m50s
-    standard (default)   kubernetes.io/host-path        8m29s
-    ```
+```
+kubectl cluster-info
+```
 
 ## Step 2: Deploy TiDB Operator
 
-Add the official chart repository provided by PingCAP:
+First, install Helm. You can either refer to [Installing Helm](https://helm.sh/docs/intro/install/) or install the latest stable version of Helm 3:
```

Comment:
Would it be better to use an ordered list?

Reply:
Updated.